src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,18 +49,51 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringListProperty Name="AvailablePlatforms" Separator="," Visible="False" />
-  <StringProperty Name="FileName" DisplayName="Project File" ReadOnly="True">
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants"/>
+    <EnumProperty Name="PlatformTarget" DisplayName="Platform Target"/>
```

Comment:
vslangproj seems to do some validation of the PlatformTarget. You probably need to intercept this property setter and do the validation. Also why is this an EnumProperty but doesn't have any values defined?